RunningOrder/Shared/Extensions/Cloudkit+Extensions.swift

```diff
@@ -132,6 +132,76 @@ extension CKFetchRecordsOperation {
     }
 }
 
+extension CKModifySubscriptionsOperation {
+    func publisher() -> AnyPublisher<([CKSubscription]?, [CKSubscription.ID]?), Error> {
+        let modifySubscriptions = PassthroughSubject<([CKSubscription]?, [CKSubscription.ID]?), Error>()
+        self.modifySubscriptionsCompletionBlock = { subscriptions, ids, error in
+            if let error = error {
+                modifySubscriptions.send(completion: .failure(error))
+                return
+            }
+
+            modifySubscriptions.send((subscriptions, ids))
+            modifySubscriptions.send(completion: .finished)
+        }
+
+        return modifySubscriptions.eraseToAnyPublisher()
+    }
+}
+
+extension CKFetchRecordZoneChangesOperation {
+    func publishers() -> (fetchRecordZoneChangesCompletion: AnyPublisher<Never, Error>,
```

Comment:
no because it is all publishers based on a single operation. Most of the times, completion of all publishers goes in the completionBlock of the operation, so not possible to change it multiple times (each method is called). Also it is not possible to add a stored publisher as it is an extension, or it could be easily removed by filling the original blocks..